docs/spelling_wordlist.txt

```diff
@@ -54,4 +56,5 @@ Versioning
 vertices
 workflow
 workflows
+zhvelo
```

Comment:
Nooooo :scream:

Reply:
I know, it felt a little bit dirty... :)